test_agent/rust/src/constants.rs

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024 General Motors GTO LLC
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ * SPDX-FileType: SOURCE
+ * SPDX-FileCopyrightText: 2023 General Motors GTO LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+pub const SEND_COMMAND: &str = "send";
+pub const REGISTER_LISTENER_COMMAND: &str= "registerlistener";
+pub const UNREGISTER_LISTENER_COMMAND: &str = "unregisterlistener";
+
+pub const RESPONSE_ON_RECEIVE: &str = "onreceive";
+
+// Define constants for addresses
+pub const DISPATCHER_ADDR: (&str, u16) = ("127.0.0.1", 44444);
+pub const TEST_MANAGER_ADDR: (&str, u16) = ("127.0.0.5", 12345);
+
+// Define constant for maximum message length
+pub const BYTES_MSG_LENGTH: usize = 32767;
+
+#[test]
+pub fn test_constants() {
+    dbg!("SEND_COMMAND: {}", SEND_COMMAND);
```

Comment:
I'd suggest you use the `log` and `env_logger` crates so that you can have different log levels and control them being turned on or off from the command line.

You then be able to from the command line set:
```
RUST_LOG=trace
```
(or debug, or warn, or whatever you choose)
to be able to get exactly that level of logging.

And if you didn't want any logging anymore you can unset the env var.

Here's [a PR](https://github.com/eclipse-uprotocol/up-streamer-rust/pull/8/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R33) in which that's used.

Reply:
Yup, still suggest this

Reply:
I see you're using the log crate's `error!()` elsewhere. Maybe migrate all the `dbg!()` over to using the log crate's `debug!()`.

Reply:
fixed